docs/security.html

```diff
@@ -847,27 +847,27 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
                 Kafka supports <a href="https://tools.ietf.org/html/rfc7677">SCRAM-SHA-256</a> and SCRAM-SHA-512 which
                 can be used with TLS to perform secure authentication. Under the default implementation of <code>principal.builder.class</code>, the username is used as the authenticated
                 <code>Principal</code> for configuration of ACLs etc. The default SCRAM implementation in Kafka
-                stores SCRAM credentials in Zookeeper and is suitable for use in Kafka installations where Zookeeper
-                is on a private network. Refer to <a href="#security_sasl_scram_security">Security Considerations</a>
+                stores SCRAM credentials in KRaft controllers. Refer to <a href="#security_sasl_scram_security">Security Considerations</a>
```

Comment:
In other places in the docs, we use "in the metadata log" to described where metadata is stored in KRaft mode. For example, see:
- https://kafka.apache.org/documentation/#brokerconfigs_early.start.listeners
- https://kafka.apache.org/documentation/#security_authz

I think we should stick with that term.

Reply:
Thanks for the suggestion. Updated related parts.